java/java.api.common/src/org/netbeans/modules/java/api/common/resources/Description.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+<html>
+    <head>
+        <title>TODO supply a title</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Does Swing actually support the meta charset tag? The viewport looks strange in this context.